File: app/src/main/java/com/statuses/statussavers/MoreActivity.java
package com.statuses.statussavers;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.browser.customtabs.CustomTabsIntent;
import androidx.cardview.widget.CardView;

import android.content.ActivityNotFoundException;
import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.view.MenuItem;
import android.view.View;

import com.google.android.gms.ads.AdRequest;
import com.google.android.gms.ads.AdView;
import com.google.android.gms.ads.MobileAds;
import com.google.android.material.bottomnavigation.BottomNavigationView;

public class MoreActivity extends AppCompatActivity {

    private BottomNavigationView bottomNavigationView;
    private CardView cardView;
   // private AdView moreAdView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_more);
        bottomNavigationView = (BottomNavigationView) findViewById(R.id.bottomNavigationViewMore);
        cardView = (CardView) findViewById(R.id.cardView);
        //moreAdView = (AdView) findViewById(R.id.moreAdView);
        setupBottomBar();
        setupCardView();
        //setbannerAd();
    }

    private void setupCardView() {
        cardView.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                String url = "https://860.game.qureka.com";
                try {
                    CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();
                    CustomTabsIntent customTabsIntent = builder.build();
                    customTabsIntent.launchUrl(MoreActivity.this, Uri.parse(url));
                }
                catch (ActivityNotFoundException e) {
                    Intent browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse(url));
                    startActivity(browserIntent);
                }
            }
        });
    }

    private void setupBottomBar() {
        bottomNavigationView.setSelectedItemId(R.id.more);
        bottomNavigationView.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
            @Override
            public boolean onNavigationItemSelected(@NonNull MenuItem item) {

                switch(item.getItemId())
                {
                    case R.id.home:
                        openHomeActivity();
                        return true;
                    case R.id.more:
                        return true;
                }
                return false;
            }
        });
    }

    private void setbannerAd() {
        MobileAds.initialize(this);
        AdRequest adRequest = new AdRequest.Builder().build();
        //moreAdView.loadAd(adRequest);
    }

    private void openHomeActivity() {
        Intent intent = new Intent(this, MainActivity.class);
        startActivity(intent);
    }

    @Override
    protected void onResume() {
        super.onResume();
        bottomNavigationView.setSelectedItemId(R.id.more);
    }
}

File: app/src/main/java/com/statuses/statussavers/MoreActivity.kt
package com.statuses.statussavers

import android.content.ActivityNotFoundException
import android.content.Intent
import android.net.Uri
import android.os.Bundle
import android.view.MenuItem
import android.view.View
import android.view.ViewGroup
import androidx.appcompat.app.AppCompatActivity
import androidx.browser.customtabs.CustomTabsIntent
import androidx.cardview.widget.CardView
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import androidx.core.view.updateLayoutParams
import androidx.core.view.updatePadding
import com.google.android.material.bottomnavigation.BottomNavigationView

class MoreActivity : AppCompatActivity() {

    private lateinit var bottomNavigationView: BottomNavigationView
    private lateinit var cardView: CardView
    private lateinit var rootLayout: View

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_more)
        rootLayout = findViewById(R.id.rootLayout)
        bottomNavigationView = findViewById(R.id.bottomNavigationViewMore)
        cardView = findViewById(R.id.cardView)

        ViewCompat.setOnApplyWindowInsetsListener(rootLayout) { _, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            rootLayout.updatePadding(top = systemBars.top)
            val layoutParams = cardView.layoutParams as ViewGroup.MarginLayoutParams
            layoutParams.setMargins(15, 180, 15, 0)
            cardView.layoutParams = layoutParams
            bottomNavigationView.updatePadding(bottom = systemBars.bottom)
            insets
        }

        setupBottomBar()
        setupCardView()
    }

    private fun setupCardView() {
        cardView.setOnClickListener {
            val url = "https://860.game.qureka.com"
            try {
                val builder = CustomTabsIntent.Builder()
                val customTabsIntent = builder.build()
                customTabsIntent.launchUrl(this@MoreActivity, Uri.parse(url))
            } catch (e: ActivityNotFoundException) {
                val browserIntent = Intent(Intent.ACTION_VIEW, Uri.parse(url))
                startActivity(browserIntent)
            }
        }
    }

    private fun setupBottomBar() {
        bottomNavigationView.selectedItemId = R.id.more
        bottomNavigationView.setOnNavigationItemSelectedListener { item ->
            when (item.itemId) {
                R.id.home -> {
                    openHomeActivity()
                    true
                }
                R.id.dm -> {
                    openDirectMessageActivity()
                    true
                }
                R.id.more -> true
                else -> false
            }
        }
    }

    private fun openDirectMessageActivity() {
        bottomNavigationView!!.selectedItemId = R.id.home
        val intent = Intent(this, DirectMessage::class.java)
        startActivity(intent)
    }

    private fun openHomeActivity() {
        val intent = Intent(this, MainActivity::class.java)
        startActivity(intent)
    }

    override fun onResume() {
        super.onResume()
        bottomNavigationView.selectedItemId = R.id.more
    }
}
